server/routes/upsertHistoricalValidatorStats.ts

```diff
@@ -0,0 +1,24 @@
+import { Request, Response, NextFunction } from 'express';
```

Comment:
We won't ever be creating these records through a route. They should all be created in the eventHandlers. Please remove this file.

Reply:
I actually wanted some data in Valdiator and HistoricalValidator table so I created those routes for development.WOuld be removed once event-handlers work is done

Reply:
Sounds good.